packages/itwin/one-click-lca-widget/package.json

```diff
@@ -43,14 +43,25 @@
     "@itwin/itwinui-react": "^1.29.2"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/build-tools": "^3.0.0",
-    "@itwin/certa": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
+    "@itwin/appui-abstract": "^3.1.2",
```

Comment:
I believe ^3.0.0 is good enough for all of these.

Reply:
Bumped it back down to 3.0.0
Not sure why, but at first rush was giving warnings about unmet peer dependencies for all these @itwin packages and the warnings went away after bumping the versions up to 3.1.2.
Now that I reverted to 3.0.0 the warnings did not reappear and everything seems to be in order.